sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_operations/_patch.py

```diff
@@ -201,9 +265,75 @@ def send_to_all(  # pylint: disable=inconsistent-return-statements
         if cls:
             return cls(pipeline_response, None, {})
 
+    @overload
+    def send_to_user(  # pylint: disable=inconsistent-return-statements
+        self, user_id: str, message: Union[str, JSON], *, filter: Optional[str] = None, content_type: Optional[str] = "application/json", **kwargs: Any
```

Comment:
same with https://github.com/Azure/azure-sdk-for-python/pull/34855/files#r1531725873